gaia_tasks/tasks.py

```diff
@@ -0,0 +1,35 @@
+import json
+import re
+from girder_worker.app import app
+from girder_worker.utils import girder_job
+from gaia.parser import deserialize
+
+
+@girder_job()
+@app.task(bind=True)
+def gaia_task(self, kwargs):
```

Comment:
Choosing to implement a single task in this way will seriously degrade the capabilities of Gaia with regard to Girder Worker.  As written the current task implements input and output concepts that are more appropriately handled as apart of celery's underlying canvas architecture (e.g. chaining,  groups,  map-reduce style chords).  The purpose of a girder worker task is the encapsulate a function that does some unit of actual work.  As written this encapsulates a function that encapsulates functions confusing the spheres of responsibility. 

It is difficult to foresee how this design choice will limit your ability to use upstream features of Girder Worker.